app/networkapi/templates/filebrowser/upload.html

```diff
@@ -101,10 +106,15 @@
             <p class="deletelink-box">
                 <a class="deletelink" href="javascript://">{% trans "Clear Queue" %}</a>
             </p>
+
+            <!-- override: start #160 -->
+            <!-- <input class="default" type="submit" name="_save" value='{% trans "Upload" %}' /> -->
```

Comment:
it's fine to do this on the same line instead of duplicating just to add an id attribute.